application/src/test/java/org/opentripplanner/model/plan/paging/cursor/PageCursorFactoryTest.java

```diff
@@ -181,30 +201,30 @@ private void assertPageCursor(
     Instant expLat,
     Duration expSearchWindow,
     PageType expPageType,
-    Boolean hasDedupeParams
+    Boolean hasDedupeParams,
```

Comment:
What does this param mean, I think it probably has a typo. I know you haven't added it but I think we can improve its name.

Reply:
I renamed it to `hasDeduplicationParams` and added a comment